### PR TITLE
[ON] Add default bundle

### PR DESCRIPTION
**Pull request by MosCD3**
Work on progress to finalize this first bundle contribution

### CODE REVIEW

File: OCABundles/credentials/ongov-digital-trust/ontario-test-credential/OCABundle.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
```
Comment (swcurran) [inline OCABundles/credentials/ongov-digital-trust/ontario-test-credential/OCABundle.json:1]: @MosCD3 — the format is not right.  The outer brackets should be square, and what you have for line 2 is not right — the CredDef ID.

File: OCABundles/credentials/ongov-digital-trust/ontario-test-credential/OCABundle.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+              "country": "Pays",
+              "postal_code": "Code Postal",
+              "birth_dateint": "Date de naissance",
+              "issue_date": "Date d'émission",
+              "expiry_date": "Expiration",
+              "test_id": "Test ID"
+            }
+          },
+          {
+            "capture_base": "EfeQBz5tyC0gett-ETgkg4pkxBwwuQoeYsT-9Nt0cNsc",
```
Comment (swcurran) [inline OCABundles/credentials/ongov-digital-trust/ontario-test-credential/OCABundle.json:65]: This is not the “branding” overlay that we have defined in Aries.  The I’m not sure if “card_overlay” is a OCA overlay or not, but it doesn’t match what is expected by Aries wallets.

**Review by swcurran — CHANGES_REQUESTED** [review]
I think the issues raised need to be addressed before merging.

File: OCABundles/credentials/ongov-digital-trust/ontario-test-credential/OCABundle.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "capture_base": {
+      "attributes": {
```
Comment (swcurran) [inline OCABundles/credentials/ongov-digital-trust/ontario-test-credential/OCABundle.json:4]: The attributes list does not match the attributes in the schema that is identified.  Expecting (based on the schema: 

          "region",
          "issued",
          "street_address",
          "birthdate",
          "family_name",
          "country",
          "postal_code",
          "locality",
          "given_names"

**Comment by swcurran** [comment]
Sorry for the delay in providing feedback — I’ve been out of the office and travelling.

I notice that you put the bundle in the “credentials” folder but the identifier references a schema object on Sovrin TestNet. As defined and based on instructions in the README, the folder should be in the `schema` folder.  If we don’t change anything, that needs to be moved before merging.

It does raise the question — should there even be a differentiation in folders?  We could make it that there is just one folder that contains the “publisher” of the referenced object.

Any thoughts on that?

**Comment by amanji** [comment]
Is there any update on this, or can we close until this is ready for PR again?

**Pull request closed without merging**